redskyctl/internal/commands/generate/experiment/generator.go

```diff
@@ -65,6 +67,18 @@ func (g *Generator) SetDefaultSelectors() {
 			g.ReplicaSelectors[i].LabelSelector = g.Application.Parameters.Replicas.LabelSelector
 		}
 	}
+
+	if g.Application.Parameters != nil && len(g.Application.Parameters.ConfigMaps) > 0 {
```

Comment:
nit, we should be able to drop `&& len(g.Application.Parameters.ConfigMaps) > 0`, the range below should account for it.

Reply:
...yeah, I only put it in because of an OCD reaction to make it look like the stuff above (where it was warranted because there is additional logic in the non-empty case).